test/unit/models/classes/Security/DataAccess/Repository/CachedPlatformJwksRepositoryTest.php

```diff
@@ -0,0 +1,128 @@
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoLti\test\unit\models\classes\Security\DataAccess\Repository;
+
+use oat\generis\test\TestCase;
+use oat\oatbox\cache\SimpleCache;
+use oat\tao\model\security\Business\Domain\Key\Jwk;
+use oat\tao\model\security\Business\Domain\Key\Jwks;
+use oat\taoLti\models\classes\Security\DataAccess\Repository\CachedPlatformJwksRepository;
+use oat\taoLti\models\classes\Security\DataAccess\Repository\PlatformJwksRepository;
+
+class CachedPlatformJwksRepositoryTest extends TestCase
+{
+    /** @var CachedPlatformJwksRepository */
+    private $subject;
+
+    /** @var SimpleCache */
+    private $cacheMock;
+
+    /** @var PlatformJwksRepository */
+    private $platformJwksRepositoryMock;
+
+    /** @var Jwks  */
+    private $jwks;
+
+    public function setUp(): void
+    {
+        $this->platformJwksRepositoryMock = $this->createMock(PlatformJwksRepository::class);
+        $this->cacheMock = $this->createMock(SimpleCache::class);
+        $this->subject = new CachedPlatformJwksRepository();
+        $this->subject->setServiceLocator($this->getServiceLocatorMock(
+            [
+                PlatformJwksRepository::class => $this->platformJwksRepositoryMock,
+                SimpleCache::SERVICE_ID => $this->cacheMock,
+            ]
+        ));
+
+        $jwk = new Jwk(
+            'kty',
+            'e',
+            'n',
+            'kid',
+            'alg',
+            'use'
+        );
+
+        $this->jwks = new Jwks($jwk);
+
+    }
+
+    public function testCachedFind(): void
+    {
+        $this->platformJwksRepositoryMock
+            ->method('find')
+            ->willReturn($this->jwks);
+
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('has')
+            ->with('PLATFORM_JWKS')
+            ->willReturn(true);
+
+
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('delete')
+            ->with('PLATFORM_JWKS');
+
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('set')
+            ->with(
+                'PLATFORM_JWKS', $this->jwks->jsonSerialize()
+            );
+
+        $this->subject->find();
+    }
+
+    public function testNotCachedFind(): void
+    {
+
+        $this->platformJwksRepositoryMock
+            ->method('find')
+            ->willReturn($this->jwks);
+
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('has')
+            ->with('PLATFORM_JWKS')
+            ->willReturn(false);
+
+
+        $this->cacheMock
+            ->expects($this->never())
+            ->method('delete')
+            ->with('PLATFORM_JWKS');
+
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('set')
+            ->with(
+                'PLATFORM_JWKS', $this->jwks->jsonSerialize()
```

Comment:
```suggestion
                'PLATFORM_JWKS', 
                $this->jwks->jsonSerialize()
```

Since you broke the parameters passing in new line, there should be one per line according to PSR-2.